demisto_sdk/commands/test_content/tests/test_context_test.py

```diff
@@ -822,9 +823,7 @@ def test_replacing_pb_inputs(mocker, current, new_configuration, expected):
     playbook_instance.integrations[0].integration_type = Docker.PYTHON_INTEGRATION_TYPE
 
     class clientMock(DefaultApi):
-        def generic_request(self, path, method, body=None, **kwargs):
-            if path == "/about" and method == "GET":
-                return "{'demistoVersion': '6.5.0'}", None, None
+        ...
```

Comment:
What are these changes meant for?

Reply:
don't need the client content since the server_version is passed to the function through the argument